src/detect-parse.c

```diff
@@ -1722,6 +1725,15 @@ static int SigValidate(DetectEngineCtx *de_ctx, Signature *s)
                 }
                 sm = sm->next;
             }
+            /* if stream_size is in use, also inspect packets */
+            sm = s->init_data->smlists[DETECT_SM_LIST_MATCH];
```

Comment:
I don't manage to see how we check that there is a raw content match. I think I've missed it but it is better to ask.

Reply:
Making this more clean in the next iteration.

Reply:
So `ignore` isn't the best term either. `default` is OK if well documented.